src/Client/Web/Bit.Client.Web.BlazorUI/Components/BitLabel.razor.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Microsoft.AspNetCore.Components;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Bit.Client.Web.BlazorUI.Components
+{
+    public partial class BitLabel
+    {
+        [Parameter] public string Text { get; set; }
+        [Parameter] public bool IsRequired { get; set; }
+        [Parameter] public string componentRef { get; set; }
```

Comment:
Could you please provide me a link that describes this approach? I mean string componentRef

Reply:
this property refers to "for" attribute of "label" element. 
Um, I think ,my naming wasn't good and made ambiguity, so I changed its name.